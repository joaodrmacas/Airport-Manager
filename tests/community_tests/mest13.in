a LIS Portugal Lisbon
a OPO Portugal Porto
v TP12 LIS OPO 23-03-2022 12:45 13:40 81
v TP12 LIS OPO 23-03-2024 09:56 00:45 79
q
