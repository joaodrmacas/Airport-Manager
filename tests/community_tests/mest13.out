airport LIS
airport OPO
invalid duration
invalid date
